codewars8.py

```diff
@@ -0,0 +1,36 @@
+# https://www.codewars.com/kata/58b3c2bd917a5caec0000017/train/python
+# works locally, but codewars returns error:/
```

Comment:
Works locally because you use only one example to test it. If you change the array to e.g. `[2, 1, 2, 2, 6, 5, 0, 2, 0, 3, 3, 3, 9, 2]`, it will fail with `TypeError: not all arguments converted during string formatting`.